to_srt.py

```diff
@@ -43,22 +46,36 @@ def append_subs(start, end, prev_content, format_time):
     # this regex was sometimes too strict. I hope the new one is never too lax
     # content_re = re.compile(u'xml\:id\=\"subtitle[0-9]+\">(.*)</p>')
     content_re = re.compile(u'\">(.*)</p>')
-    alt_content_re = re.compile(u'<span style=\"[a-zA-Z0-9_]+\">(.*?)</span>')
+	# span tags are only used for italics, so we'll get rid of them
+	# and replace them by <i> and </i>, which is the standard for .srt files
+    # alt_content_re = re.compile(u'<span style=\"[a-zA-Z0-9_]+\">(.*?)</span>')
+    span_start_re = re.compile(u'(<span style=\"[a-zA-Z0-9_.]+\">)+')
+    span_end_re = re.compile(u'(</span>)+')
     br_re = re.compile(u'(<br\s*\/?>)+')
     fmt_t = True
     for s in sub_lines:
+        span_start_tags = re.search(span_start_re, s)
+        if span_start_tags:
+            s = u"<i>".join(s.split(span_start_tags.group()))
+        string_region_re = r'<p(.*region="' + xml_id_display_align_before + r'".*")>(.*)</p>'		
+        #s = re.sub(r'<p(.*region="region.top.centre".*")>(.*)</p>', r'<p\1>{\\an8}\2</p>', s)

```

Comment:
What does this comment stand for?

Reply:
It was just for testing purposes, you can remove it ;)